packages/url/src/index.js

```diff
@@ -18,6 +18,146 @@ export function isURL( url ) {
 	return URL_REGEXP.test( url );
 }
 
+/**
```

Comment:
We still need to add documentation for newly introduced functions: https://github.com/WordPress/gutenberg/blob/6a0c4441d4f7eb8dd90fb88f0aeec18291170e3a/packages/url/README.md

We also need to announce minor version increment for the `url` package with the list of all methods introduced.

Reply:
👍 Now done